Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs

```diff
@@ -0,0 +1,349 @@
+using System;
+using UniRx;
+using UnityEngine;
+using WhereIsMyWife.Controllers;
+using WhereIsMyWife.Managers.Properties;
+using WhereIsMyWife.Player.State;
+using Zenject;
+
+namespace WhereIsMyWife.Managers
+{
+    public partial class PlayerManager 
+    {
+        [Inject] private IPlayerProperties _properties;
+
+        [Inject] private IRunningMethods _runningMethods;
+        [Inject] private IJumpingMethods _jumpingMethods;
+        
+        // Timers
+        private float _lastOnGroundTime = 0;
+        private float _lastPressedJumpTime = 0;
+    }
+    
+    public partial class PlayerManager : IPlayerStateIndicator
```

Comment:
Just make it Monobehavior.

Reply:
209e6df3a42a40a8862112dec422e6301cb8a73e